sdks/python/apache_beam/examples/datastore_wordcount.py

```diff
@@ -61,53 +65,134 @@ def process(self, context):
     return words
 
 
+class CreateEntityFn(object):
+  """Create a Cloud Datastore entity from the given string."""
+  def __init__(self, namespace, kind, ancestor):
+    self._namespace = namespace
+    self._kind = kind
+    self._ancestor = ancestor
+
+  def make_entity(self, content):
+    entity = entity_pb2.Entity()
+    if self._namespace is not None:
+      entity.key.partition_id.namespace_id = self._namespace
+
+    # All entities created will have the same ancestor
+    datastore_helper.add_key_path(entity.key, self._kind, self._ancestor,
+                                  self._kind, str(uuid.uuid4()))
+
+    datastore_helper.add_properties(entity, {"content": unicode(content)})
+    return entity
+
```

Comment:
I think it is needed for global functions. pylint complains. apache_beam/examples/datastore_wordcount.py:87:1: E302 expected 2 blank lines, found 1